docs/concepts/event-subsriber.md

```diff
@@ -8,4 +8,20 @@ type: markdown
 ---
 <h2 class="top">Event Subscriber</h2> 
 
-Event Subscriber is an object that is subscribed to receive events.
\ No newline at end of file
+Event Subscriber is an object that is subscribed to receive events.
```

Comment:
This is an example of command handling, not event subscribing.

Reply:
LGTM
I will place this example to the `Command Handler` section then.